go/vt/mysqlctl/replication.go

```diff
@@ -69,8 +74,8 @@ func changeMasterArgs(params *sqldb.ConnParams, masterHost string, masterPort in
 }
 
 // parseSlaveStatus parses the common fields of SHOW SLAVE STATUS.
-func parseSlaveStatus(fields map[string]string) *proto.ReplicationStatus {
-	status := &proto.ReplicationStatus{
+func parseSlaveStatus(fields map[string]string) proto.ReplicationStatus {
```

Comment:
To be consistent with proto.ReplicationPosition everywhere. It's a small structure anyway, that has one interface and 5 simple-type members.